onnxscript/tests/function_libs/torch_lib/ops_test_common.py

```diff
@@ -81,8 +82,10 @@ def xfail(
     *,
     reason: str,
     dtypes: Optional[Collection[torch.dtype]] = None,
+    matcher: Optional[Callable[[Any], Any]] = None,
     enabled_if: bool = True,
     test_class_name: Optional[str] = None,
+    name: str = "xfail",
```

Comment:
Is this duplicated information since we know it’s xfail already from the function?

Reply:
You are right. This could use decorator_meta.name.

Reply:
How do I get function name in this context?